packages-exp/auth-exp/src/index.ts

```diff
@@ -15,9 +15,42 @@
  * limitations under the License.
  */
 
+import * as externs from '@firebase/auth-types-exp';
+import { ErrorFn, CompleteFn, Unsubscribe } from '@firebase/util';
+
 // core/auth
 export { initializeAuth } from './core/auth/auth_impl';
 
+// Non-optional auth methods.
```

Comment:
I liked having the index.ts file here be 100% re-exported functions. WDYT about moving these methods into `src/auth/util.ts` or something along those lines?

Reply:
possibly! let's hold off until we write the polyfill, I imagine we will have a lot of refactoring then